css/at-rules/document.json

```diff
@@ -34,25 +34,11 @@
                 "version_removed": "61"
               }
             ],
-            "firefox_android": [
-              {
-                "prefix": "-moz-",
-                "version_added": "61",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "layout.css.moz-document.content.enabled",
-                    "value_to_set": "true"
-                  }
-                ],
-                "notes": "Disabled by default in web pages, except for an empty <code>url-prefix()</code> value, which is supported due to its <a href='https://css-tricks.com/snippets/css/css-hacks-targeting-firefox/'>use in Firefox browser detection</a>. Still supported in user stylesheets."
```

Comment:
This note suggests there's still partial support. Can you investigate and rewrite the note if needed?